roles/ceph-common/tasks/checks/check_firewall.yml

```diff
@@ -1,6 +1,6 @@
 ---
 - name: check if nmap is installed
-  command: "command -v nmap"
+  local_action: shell command -v nmap
```

Comment:
My first draft of this PR used the command module and it was failing, hence my use of shell instead. Testing it again now shows that it works...I can only blame myself :(